package.json

```diff
@@ -45,8 +60,8 @@
     "serverless-stack-termination-protection": "^2.0.2"
   },
   "engines": {
-    "node": "^16",
-    "npm": "^8"
+    "node": "^18",
```

Comment:
@NicholasValiente I noticed that the serverless.yml 
provider:
  logRetentionInDays: 30
  name: aws
  `runtime: nodejs16.x`

Reply:
fixed, thanks for pointing that out 👍